topdown/semver.go

```diff
@@ -0,0 +1,56 @@
+package topdown
```

Comment:
nit: Please add the OPA copyright header to these new files in topdown

Reply:
Oops, should be all present and correct now.